extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -284,21 +264,19 @@ public void testDateTimeChannelCalendarDefaultTimeZoneUpdate() {
         // Initialize with configuration with no time zone property set.
         initialize(configuration, NtpBindingConstants.CHANNEL_DATE_TIME, ACCEPTED_ITEM_TYPE_DATE_TIME, null, null);
 
-        String timeZoneIdFromItemRegistry = ((DateTimeType) getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME)).getCalendar()
-                .getTimeZone().getID();
-
+        ZoneOffset timeZoneIdFromItemRegistry = ((DateTimeType) getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME))
+                .getZonedDateTime().getOffset();
         assertThat("The dateTime channel calendar was not updated with the right timezone", timeZoneIdFromItemRegistry,
-                is(equalTo(DEFAULT_TIME_ZONE_ID)));
+                is(equalTo(ZoneOffset.of("+03:00"))));
     }
 
     @Test
     public void testStringChannelFormatting() {
-        final String formatPattern = "EEE, d MMM yyyy HH:mm:ss Z";
+        final String formatPattern = "EEE, d MMM yyyy HH:mm:ss z";
```

Comment:
what was wrong about the `Z`?

Reply:
It's a matter of representation. I just adapted the pattern to the changes. [Corresponding oracle documentation](https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html).